ext/intl/spoofchecker/spoofchecker_main.c

```diff
@@ -143,8 +143,8 @@ PHP_METHOD(Spoofchecker, setRestrictionLevel)
 			USPOOF_MODERATELY_RESTRICTIVE != level &&
 			USPOOF_MINIMALLY_RESTRICTIVE != level &&
 			USPOOF_UNRESTRICTIVE != level) {
-		php_error_docref(NULL, E_WARNING, "Invalid restriction level value");
-		return;
+		zend_argument_value_error(1, "must be one of the Spoofchecker::*_RESTRICTIVE constants or Spoofchecker::UNRESTRICTIVE");
```

Comment:
Seems tricky :) In this case, you could just list them all, 4 items don't seem too many for me. If the list gets too long, then just write "is invalid". Problem solved! :D